runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -51,6 +63,17 @@
 public class KingdomPlugin extends Plugin
 {
 	private static final ImmutableSet<Integer> KINGDOM_REGION = ImmutableSet.of(10044, 10300);
+	private static final String CONFIG_LAST_CHANGED_KEY = "lastChanged";
+	private static final String CONFIG_COFFER_KEY = "coffer";
+	private static final String CONFIG_FAVOR_KEY = "favor";
+	private static final String CHAT_MESSAGE_FORMAT = "Your Kingdom of Miscellania favor is %d%%, and your coffer has %s coins.";
```

Comment:
why isn't this `%d coins`?

Reply:
It prints as a string like `1.16M`, not an int like`1160395`

Reply:
Oh you're right. This seems fine then, but I think "favor" is actually "approval rating" (I cannot find any use of favor/favour to describe it anywhere). Might be best to change this here and in the configs etc.